docs/omg-history.md

```diff
@@ -0,0 +1,143 @@
+today, both agreed in principle to join together under the SPDX 3.0 label but
+to make several changes in the way the SPDX community activities were run as
+well as how the resulting specification would be vetted.
+
+![Figure 15 – Correspondence between 3T-SBOM and SPDX 3.0 draft models (circa Sep 2020)](images/3tsbom-spdx30-correspondence.png "Figure 15 – Correspondence between 3T-SBOM and SPDX 3.0 draft models (circa Sep 2020)")
+**Figure 15 – Correspondence between 3T-SBOM and SPDX 3.0 draft models (circa
+Sep 2020)**
+
+Specifically, the SPDX community revised their charter to align with the
+processes of a Standards Development Organization, electing new chairs and
+adding the OMG Architecture Board review as a gating factor in the publication
+of SPDX 3.0 and subsequent versions.
+
+The merged activities of the two group slid together the beginning weeks of
+2021 with activities generally moving forward but occasionally stalling while
+the larger group worked through issues that one or the other hadn’t discussed
+or had a different opinion about. Eventually, after releasing SPDX 2.3 in
+August of 2022 with updates that brought some of the concepts and capabilities
+slated for SPDX 3.0 to the community in preparation of the shift that SPDX 3.0
+represents, the first release candidate of SPDX 3.0 was released in May of
+2023. Within the SPDX community, which is both a standards creation
+organization as well as a community of open source developers, a release
+candidate offers an opportunity for implementors of SPDX, both new and old, to
+review the work and determine whether there were parts that were unclear or
+that would be extremely burdensome to implement.
+
+Based on the comments and change requests from the initial candidate release
+several areas of the model were revised and reworked, resulting in a release
+candidate 2 of SPDX 3.0 in February of 2024. This release candidate will give
+tool creators and those who maintain the support libraries for working with
+SPDX time to start revising their projects in advance of the final version of
+the specification. For those not following the inner workings, debates, and
+discussion of the combined 3T-SBOM and SPDX 3.0 working group for the last 3
+years there will be a dramatic change in the SPDX model as it goes from SPDX
+2.3 to SPDX 3.0, as shown by looking at Figure 16’s left-side (SPDX 2.3 model)
+compared to its right-side (SPDX 3.0), shifting the SPDX name from Software
+Package Data Exchange to System Package Data Exchange and the scope of items it
+can convey in a bill of materials.
+
+![Figure 16 – SPDX 2.3 Model compared to the SPDX 3.0 Model](images/spdx23-spdx30-comparison.png "Figure 16 – SPDX 2.3 Model compared to the SPDX 3.0 Model")
+**Figure 16 – SPDX 2.3 Model compared to the SPDX 3.0 Model**
+
+The SPDX 3.0.1 model is available at:
+[https://github.com/spdx/spdx-3-model](https://github.com/spdx/spdx-3-model)
```

Comment:
SHould this be 3.0.1?

Reply:
I'm not sure. @goneall this one supposed to be a model file/RDF?

Reply:
Should it be https://spdx.github.io/spdx-spec/v3.0.1/rdf/schema.json ?

(That URL has been used in "5.12 Standard data format requirements" section of Conformance - https://github.com/spdx/spdx-spec/pull/1046, but the section has recently been removed.)

Reply:
It is now read

```Markdown
The SPDX 3.0.1 model is available at:
[https://spdx.github.io/spdx-spec/v3.0.1/rdf/schema.json](https://spdx.github.io/spdx-spec/v3.0.1/rdf/schema.json)
```

Reply:
I just checked and the schema is not being published there.

@zvr - do you happen to know where the schema is published?  On release, we manually copy it over to the spdx.dev website, but I don't know if there is a location where it is automatically generated.

Reply:
@goneall for v3.0 it is at https://spdx.github.io/spdx-spec/v3.0/model/schema.json

But the new spec-parser will publish v3.0.1 one at https://spdx.github.io/spdx-spec/v3.0.1/rdf/schema.json

See @zvr comment here: https://github.com/spdx/spdx-spec/pull/1046#discussion_r1713022625

Reply:
I decided to remove that three lines.
The model URL will be in the Serialization chapter, per this PR #1059